SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005808	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Duration: #{CaptionTime}#second{nl}#{CaptionRatio2}#Additional damage per second: 
+SKILL_20151224_005809	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon evil spirits from the death of enemies. Summoned spirits remain in the field damage enemies in contact with them.
+SKILL_20151224_005810	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon spirits to protect the caster by surrounding them.
+SKILL_20151224_005811	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Duration: 30 second
+SKILL_20151224_005812	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon demons of hatred to attack enemies. Prioritize the enemy hit by this magic and hit with additional damage by holy attacks.
+SKILL_20151224_005813	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Stigma duration: 5 second
+SKILL_20151224_005814	Evil Sacrifice
+SKILL_20151224_005815	{#993399}{ol}[Magical]{/}{/}Direct spirits to a location, and dedicate enemies in range as a sacrifice. Summoned spirits will move towards that area.
+SKILL_20151224_005816	Install a shield to protect themselves from enemies and long-ranged attacks. Enemies without a target will attack the pavise instead.
+SKILL_20151224_005817	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Throw count: #{CaptionRatio2}#{nl}Caltrop duration: #{CaptionTime}#second{nl}Decrease enemy moving speed by 10 for 5 second
+SKILL_20151224_005818	Pick up stone ammunition used for Stone Shot.
+SKILL_20151224_005819	Attack with a crossbow in while moving in fast speed. Attacking speed is raised while additional damage is applied to basic attacks.
+SKILL_20151224_005820	Duration: #{CaptionTime}#second{nl}Increase attacking speed by 30%{nl}Additional damage: #{CaptionRatio}#%
+SKILL_20151224_005821	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Explosion count: #{CaptionRatio}#
+SKILL_20151224_005822	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Install count: #{CaptionRatio}#
+SKILL_20151224_005823	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionTime}#%{nl}Arrow count: #{CaptionRatio}#{nl}Install range: #{CaptionRatio2}#{nl}Consume 2 wires
+SKILL_20151224_005824	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}The companion bites and hold the enemy in place. The bitten enemy will have their physical defense reduced while critical hit rate bonus is applied against the enemy.
+SKILL_20151224_005825	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}The companion bites an airborne enemy and lower them to the ground.
+SKILL_20151224_005826	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}The companion bites and drags the enemy towards their owner.
+SKILL_20151224_005827	Awareness of hidden enemy count: #{CaptionRatio}#{nl}Hounding AI duration: 90 second
+SKILL_20151224_005828	Cure a poisoned comrade. If the poison is higher rank than the antidote, duration of the poison is reduced instead.
```

Comment:
Cures a poisoned ally. (or: Detoxifies a poisoned ally - Since the skill is named Detoxify. Your choice)

If the poison is higher rank than the antidote, duration of the poison is reduced instead. -> If the poison is of a higher rank than the antidote, the duration will be reduced instead.